src/clj_sqs_extended/core.clj

```diff
@@ -72,43 +73,58 @@
   (.deleteQueue sqs-client url))
 
 (defn send-message
-  [sqs-client url message]
-  (.sendMessage sqs-client url message))
+  ([sqs-client url body]
+   (send-message sqs-client url body {}))
+  ([sqs-client url body
+    {:keys [format]
+     :or   {format :transit}
+     :as   opts}]
+   (let [payload (serdes/serialize body format)]
+     (.sendMessage sqs-client url payload))))
 
 (defn send-fifo-message
-  [sqs-client url message group-id]
-  (let [request (SendMessageRequest. url message)]
-    ; WATCHOUT: The group ID is mandatory when sending fifo messages.
-    (doto request (.setMessageGroupId group-id))
-    (.sendMessage sqs-client request)))
+  ([sqs-client url body group-id]
+   (send-fifo-message sqs-client url body group-id {}))
+  ([sqs-client url body group-id
+    {:keys [format]
+     :or   {format :transit}
+     :as   opts}]
+   (let [payload (serdes/serialize body format)
```

Comment:
minor style note, since `payload` isn't used, why not `->>` this through to `SendMessageRequest.`?